webrtc.html

```diff
@@ -509,33 +509,43 @@
 
         <ol>
           <li>
-            <p>Append an object representing the current call being handled
-            (i.e. function name and corresponding arguments) to the
-            <code>operations</code> array.</p>
+            <p>If <var>connection</var>'s <a href=
+              "#dom-peerconnection-signaling-state"><code>RTCPeerConnection</code>
+              signalingState</a> is <code>closed</code>, throw an
+              <code>InvalidStateError</code> exception and abort these steps.</p>
+          </li>
+
+          <li>
+            <p>Let <var>p</var> be the promise that the called
+            function will return.
+            </p>
+          </li>
+            <p>Add <code>p</code> to <var>operations</var>.</p>
           </li>
 
           <li>
-            <p>If the length of the <code>operations</code> array is exactly 1,
-            execute the function from the front of the queue
-            asynchronously.</p>
+            <p>If the length of <var>operations</var> is exactly 1,
+            execute the steps necessary to resolve <code>p</code>.</p>
           </li>
 
           <li>
-            <p>When the asynchronous operation completes (either successfully
-            or with an error), remove the corresponding object from the
-            <code>operations</code> array. After removal, if the array is
-            non-empty, execute the first object queued asynchronously and
-            repeat this step on completion.</p>
+            <p>When a promise in <code>operations</code> is resolved,
+            remove it from <code>operations</code>.  After removal,
+            if <code>operations</code> is non-empty, execute the steps
+            necessary to resolve the first promise
+            in <code>operations</code>.
           </li>
         </ol>
 
         <p>The general idea is to have only one among <code>createOffer</code>,
-        <code>setLocalDescription</code>, <code>createAnswer</code> and
-        <code>setRemoteDescription</code> executing at any given time. If
-        subsequent calls are made while one of them is still executing, they
-        are added to a queue and processed when the previous operation is fully
-        completed. It is valid, and expected, for normal error handling
-        procedures to be applied.</p>
+        <code>setLocalDescription</code>, <code>createAnswer</code>,
+        <code>setRemoteDescription</code>,
+        and <code>addIceCandidate</code> executing at any given
+        time. If subsequent calls are made while one of them is still
+        executing, they are added to a queue and processed when the
+        previous operation is fully completed. It is valid, and
+        expected, for normal error handling procedures to be
+        applied.</p>
```

Comment:
I removed it and replace it with more precise language involving promises, which I hope you will like.